scripts/testsuite-filter

```diff
@@ -128,7 +128,9 @@ def read_white_lists(white_list_files, is_gcc):
                         print ("Each line must contail <STATUS>: .*")
                         print ("e.g. FAIL: g++.dg/pr83239.C")
                         print ("Or starts with # for comment")
-                    white_lists[key] = l
+		    if key not in white_lists:
```

Comment:
The indentation looks a little off here but very minor problem.

Reply:
Oops, I use the tab here accidentally..